data/transport missions.txt

```diff
@@ -1547,6 +1547,108 @@ mission "Shady passenger transport 3"
+
+	on accept
+		dialog `The mercenaries (or gang members, or fleeing bank robbers, or who knows what else) make themselves at home on your ship, and begin drinking and arguing. You wonder what you've gotten yourself into...`
+		"rejected illegal jobs" --
+
+	on decline
+		"rejected illegal jobs" ++
+
+	on enter
+		conversation
+			`You'd begun to doubt the wisdom of your decision to transport this band of cutthroats from the moment you agreed to it, but the money was too good to resist. You settle into the captain's chair and prepare for the voyage ahead.`
+			`	But something feels wrong. You expect to hear the sounds of drunkenness and violence coming from the passenger compartment, but instead a deathly silence fills the air. You whirl around in your seat and come face-to-face with the entire gang. They're all armed. And their weapons are pointing at you. Your heart sinks.`
+			`	Their leader appears to be a lithe and dangerous-looking young woman who holds an enormous shotgun that's almost as big as her. "Looks like we boarded the ship of fools, and here's the biggest one!" she says mockingly. "We didn't have a credit to our names, but soon we'll own your ship. Hand over the access codes and we'll let you live."`
+			choice
+				`(Shoot first!)`
+					goto shoot
+				`(Negotiate.)`
+					goto negotiate
+				`(Surrender.)`
+					goto betrayed
+
+			label shoot
+			branch "shoot and die" "shoot and live"
+				random < 75
+
+			label "shoot and die"
+			`That turns out to be a really bad decision. It's also your last. A truly excessive barrage of projectiles ends your life.`
+				die
+
+			label "shoot and live"
+			`Somehow, miraculously, you manage to reach your gun before she fires. Perhaps it's the sheer brazenness of it--no one would expect someone staring down the barrels of half a dozen guns to actually draw against them. Nevertheless, you do, and it works.`
+			`	Your first shot finds its mark, and the gang leader crumples to the floor. The other thugs shriek in alarm and begin to spray gunfire in all directions, but it's ineffective panic fire. Taking cover behind the armored captain's chair, you dispatch two more with well-aimed shots.`
+			`	The remaining three goons attempt to flee, but your fingers find the button controlling the bridge door and it slams down before they can escape. They throw down their weapons and surrender. You toss them into the brig and decide that it might be a hoot to take them to their original destination as prisoners instead of passengers, and see if you can get a bounty for bringing them in.`
+
```

Comment:
should there be a goto here? seems like it will just lead into the negotiation text